src/DoctrineBatchUtils/BatchProcessing/SimpleBatchIteratorAggregate.php

```diff
@@ -111,7 +111,7 @@ public function getIterator()
      *
      * @param Traversable            $resultSet
      * @param EntityManagerInterface $entityManager
-     * @param                        $batchSize
+     * @param int                    $batchSize
      */
     private function __construct(Traversable $resultSet, EntityManagerInterface $entityManager, $batchSize)
```

Comment:
Can we add a type declaration here?

Reply:
I do that for a while now. Works great with PhpStorm for example.

Reply:
Is it not PHP 7.2 only?

Reply:
Hmm, did I not restrict this library to PHP 7.x?

Reply:
Nope, I confuse with other feature. It should work.